script_library/run-ansible.sh

```diff
@@ -35,6 +35,9 @@ function run_ansible(){
         echo "Inventory directory (${inventorydir}) exists, adding it to the ansible-playbook call."
         inventory="-i ${inventorydir}"
     fi
+    if [[ -f ${HOME}/.socok8svenv/ara.rc ]]; then
+        source ${HOME}/.socok8svenv/ara.rc
```

Comment:
can we add an `ECHO` statement here so we are explicit that we are loading it?

Reply:
Done.